config/tests/testgrids/config_test.go

```diff
@@ -55,6 +55,7 @@ var (
 	}
 	orgs = []string{
 		"conformance",
+		"kops",
```

Comment:
I take it we have to do this to create the dashboards without the prefix?  I'm not sure how controversial this is going to be...

I wonder if we could go into companies instead?  I feel like we can justify being on par with those (e.g. istio).

Reply:
cc @BenTheElder what do you think here?